docs/Documentation/Environment/Building_Packages/acquire.md

```diff
@@ -1,35 +1,34 @@
 ---
 layout: default
-title: Acquire
+title: Getting the package
 parent: Building Packages
 grand_parent: Intermediate
 ---
 
-## Getting the package
+# Getting the package
 
 1. Change working directory to the location where you'll build the package. A convenient location is `/scratch/$USER`, which we'll use for this example. `cd /scratch/$USER`
 
 2. OpenMPI can be found at [https://www.open-mpi.org/software/ompi/](https://www.open-mpi.org/software/ompi/). This will automatically redirect you to the latest version, but older releases can be seen in the left menu bar. For this, choose version 4.1.
 
-3. There are several packaging options. Here, we'll get the bzipped tarball `openmpi-4.1.0.tar.bz2`. You can either download it to a local machine (laptop) and then scp the file over to Eagle, or get it directly on Eagle with wget.
+3. There are several packaging options. 
+Here, we'll get the bzipped tarball `openmpi-4.1.0.tar.bz2`. 
+You can either download it to a local machine (laptop) and then `scp` the file over to Kestrel/Eagle, or get it directly on Kestrel/Eagle with `wget`.
```

Comment:
Could probably make this page all generic, instead of "Eagle/Kestrel" just "HPC system" or something similar